tests/scss/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -123,6 +123,130 @@ printWidth: 80
 ================================================================================
 `;
 
+exports[`between-decl.scss format 1`] = `
+====================================options=====================================
+parsers: ["scss"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+selector {
+prop: // comment
+value;
+
+prop: /* block */ value;
+
+prop
+: value;
+}
+
+// #5603
+.grid {
+        grid-template-areas: //
+          "header header header" //
+          "sidebar content content" //
+          "footer footer footer";
+
+        grid-template-areas: 
+          "header header header" //
+          "sidebar content content" //
+          "footer footer footer";
+ }
+
+// #8052
+$font-family-rich:
+  // custom
+  'Noto Sans TC', 'Noto Sans SC', 'Noto Sans JP', 
+  // Safari for OS X and iOS (San Francisco)
+  -apple-system, BlinkMacSystemFont,
+  // fallback
+  Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif,
+  // emoji
+  'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol' !default;
+
+// TODO: make these pretty
+selector {
+prop: 
+/* block */
+value;
+
+prop
+// inline
+: 
+value;
+
+prop/* block */: 
+value;
+
+prop
+/* block */
+: 
+value;
+
+prop/* before */: // after
+value;
+
+
+prop/* before */: /* after*/
+value;
+}
+=====================================output=====================================
+selector {
+  prop: // comment
+    value;
+
+  prop: /* block */ value;
+
+  prop: value;
+}
+
+// #5603
+.grid {
+  grid-template-areas: //
+    "header header header" //
+    "sidebar content content" //
+    "footer footer footer";
+
+  grid-template-areas: "header header header" //
+    "sidebar content content" //
+    "footer footer footer";
+}
+
+// #8052
+$font-family-rich:
+  // custom
+  "Noto Sans TC", "Noto Sans SC", "Noto Sans JP",
+  // Safari for OS X and iOS (San Francisco)
+    -apple-system,
+  BlinkMacSystemFont,
+  // fallback
+    Roboto,
+  "Helvetica Neue", Helvetica, Arial, sans-serif,
+  // emoji
+    "Apple Color Emoji",
+  "Segoe UI Emoji", "Segoe UI Symbol" !default;
```

Comment:
I know this still ugly, but stable even worse

**Prettier 2.0.5**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEASAZtGBadBDAWwEsAbAT2wCciwALJAHSgAJmB6N5sAVwGcYIBJqwDkAOQgDmAZTxRezACoBhEQBpm4yRBlyF01Rq1TZ85gCkACuubD2nWfmrNMlZgHlpzABrM5AE2YiT2YAClNmADFKOTAiXkgASjtsPAAHNJI4bF4yfjgCDQAhEiIoAGsAWTwwaTz4AkisNTsOFzwSEgAjGvK7ACUILu0jAAk4EgA3OBgaPGYxOG44G3GpmbmNAEFqDo1ePRy4anQWlntmAogAKyI7ES2MrOZlCBIINwBRAhuiGxFpHAAOYQODMACqAElmN9fv9ASCwVCZGQCMMSCJmABCfxwfDcEgwADcIDUIAgaVm0F4yFAeEolAgAHdLPSEDSUB0mXg8qSQF0YmByjNpGkamUgcgYJRlmTaDACCQAOq0IjwXhisBwQHyNVESZqsjIcC8Glksq8Y4wSwxIEEPDIfAkS1k668AAeRUFwpgsgIcAAMmU4I6Oi6QG73dIJVkAIrcSQhpBO8NiyiWyjGhKmvlpaiwJVEfwwWjIAAcAAYyXmIJalTE0sa83AM9M+QBHBPwG0UjkgPC8bBQOBwXH+PmUOCdoiTm14O0O5NhuBky3EKUylcgXgxuDxxOh51bmB4LqF4ulpAAJjJ0rwpAlrwI9uNLYArHy+HBFKeOSmt5MyyQlAuKwNIYDUJSWwgdIMBkFkh6WgAvkhQA)
```sh
--parser scss
```

**Input:**
```scss
$font-family-rich:
  // custom
  'Noto Sans TC', 'Noto Sans SC', 'Noto Sans JP', 
  // Safari for OS X and iOS (San Francisco)
  -apple-system, BlinkMacSystemFont,
  // fallback
  Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif,
  // emoji
  'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol' !default;
```

**Output:**
```scss
$font-family-rich:
  // custom "Noto Sans TC", "Noto Sans SC", "Noto Sans JP",
  // Safari for OS X and iOS (San Francisco)
    -apple-system,
  BlinkMacSystemFont,
  // fallback
    Roboto,
  "Helvetica Neue", Helvetica, Arial, sans-serif,
  // emoji
    "Apple Color Emoji",
  "Segoe UI Emoji", "Segoe UI Symbol" !default;

```